src/features/collections/data_products/TaxaCount.tsx

```diff
@@ -29,10 +37,45 @@ export const TaxaCount: FC<{
     setRank(opts?.[opts.length - 1]);
     return opts;
   }, [ranksQuery.data]);
+  const rankSelId = useId();
 
   const matchId = useMatchId(collection_id);
   const selectionId = useSelectionId(collection_id);
 
+  // Sort Order
+  const sortOptions: SelectOption[] = useMemo(() => {
+    return [
```

Comment:
Let me know if there is better label text for these or the dropdown labels below

Reply:
Those look good to me